engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Array.java

```diff
@@ -26,7 +27,7 @@ public final class Array implements TruffleObject {
    *
    * @param items the element values
    */
-  @Builtin.Method(expandVarargs = 4, description = "Creates an array with given elements.")
+  @Builtin.Method(expandVarargs = 4, description = "Creates an array with given elements.", owner = Owner.MODULE)
```

Comment:
I imagine we will want to have these as static methods on a type in the longer-term, we just need to finalize the `Standard.Base` statics refactor. Right @jdunkerley ?
But I assume it will not be a problem to just switch this definition to `Owner.TYPE`?

Reply:
If you want these to be statics on a type then you a) make sure that there is a `self` parameter b) drop `owner = Owner.MODUL` bit (`Owner.TYPE` is the default).